src/useDCPWorker.tsx

```diff
@@ -71,81 +45,21 @@ declare interface IWorkerOptions {
   };
   computeGroups?: Array<any>;
   jobAddresses?: Array<string>;
+  cores?: {
+    cpu?: number;
+    gpu?: number;
+  };
   maxWorkingSandboxes?: number | undefined;
-  paymentAddress?: Address | string | null;
+  paymentAddress?: any;
```

Comment:
**question:** Why loosen the type?

i.e., were there issues with `Address | string | null`?

Reply:
I ended up removing a bunch of the types, including `Address`, since it felt like more maintenance to ensure the types were up to date. Since the `Worker`, `Supervisor` and events changed in the `dcp-client` library I had to also reflect those changes here. In the end I thought having the types weren't providing much value, so I made it easier and removed them (perhaps this was a lazy decision). Ill bring back `Address` in this case as that type isn't subject to change much, if at all ...